sql/resourceloaderarticlesPriorityMigration.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE resourceloaderarticles
```

Comment:
```suggestion
ALTER TABLE /*_*/resourceloaderarticles
``` This adds the correct table prefix